test/robot/web/client/client.go

```diff
@@ -319,6 +319,31 @@ func robotEntityLink(path string, s interface{}) interface{} {
 	return a
 }
 
+func robotTextPreview(path string, s interface{}) interface{} {
+	id := s.(string)
+
+	div := dom.NewDiv()
+	div.Element.Style.MaxWidth = 600
+	div.Element.Style.MaxHeight = 420
```

Comment:
why not the full height of the window?

Reply:
mostly because the maxheight property takes an int so I couldn't put in "100%". Secondly because my experience with css has been very utilitarian and I just wanted to make sure it didn't scroll off the end of the screen.